controllers/componentdetectionquery_controller.go

```diff
@@ -266,7 +267,9 @@ func (r *ComponentDetectionQueryReconciler) Reconcile(ctx context.Context, req c
 		log.Info(fmt.Sprintf("Deleting finished ComponentDetectionQuery resource %v", req.NamespacedName))
 		if err = r.Delete(ctx, &componentDetectionQuery); err != nil {
 			// Delete failed. Log the error but don't bother modifying the resource's status
-			log.Error(err, fmt.Sprintf("Unable to delete the ComponentDetectionQuery resource %v", req.NamespacedName))
+			logutil.LogAPIResourceChangeEvent(log, componentDetectionQuery.Name, "ComponentDetectionQuery", logutil.ResourceDelete, err)
+		} else {
+			logutil.LogAPIResourceChangeEvent(log, componentDetectionQuery.Name, "ComponentDetectionQuery", logutil.ResourceDelete, nil)
 		}
```

Comment:
Can we also log events that call spi.DownloadDevfileUsingSPI since we are initiating the call to access an end users' private repo

Reply:
we can, but I'm not sure if it's a resource change event we want to monitor.

Reply:
It's not a change event but we are calling a repo that we have privileged access too.  Maybe a new action called `ACCESS` would be appropriate?